gamification/tests/test_models.py

```diff
@@ -341,3 +349,48 @@ def test_add_badges_method(self):
 
         # After applying add_badge method
         self.assertEquals(test_participant.badges.count(), 0)
+
+    def test_update_participants_data_with_meta_review(self):
```

Comment:
Can you split this long test case to several smaller test cases?

Reply:
Can you please throw some light on how you want me to split it? It's because all of the 3 participants here are for the same stuff so it's becoming difficult to identify which form do I split it in. @li-boxuan